<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIThreadAdder</class>
 <widget class="QDialog" name="UIThreadAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="8">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="group">
     <property name="title">
      <string>Save Images to</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="leSavepath">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnChoosePath">
        <property name="toolTip">
         <string>Choose download folder</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/resources/search.png</normaloff>:/icons/resources/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbRescan">
       <property name="text">
        <string>Rescan Thread</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cbOriginalFilename">
     <property name="text">
      <string>Save with original filename</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbStartImmediately">
     <property name="text">
      <string>Start immediately</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="text">
        <string>Select all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectNone">
       <property name="text">
        <string>Select none</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnStartSelected">
       <property name="text">
        <string>Start selected</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/yes.png</normaloff>:/icons/resources/yes.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartAll">
       <property name="text">
        <string>Start all</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/media-playback-start.png</normaloff>:/icons/resources/media-playback-start.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btClearSelected">
       <property name="text">
        <string>Clear selected</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/close.png</normaloff>:/icons/resources/close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearList">
       <property name="text">
        <string>Clear all</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/no.png</normaloff>:/icons/resources/no.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClearList</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelectAll</sender>
   <signal>clicked()</signal>
   <receiver>UIThreadAdder</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelectNone</sender>
   <signal>clicked()</signal>
   <receiver>UIThreadAdder</receiver>
   <slot>selectNone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartSelected</sender>
   <signal>clicked()</signal>
   <receiver>UIThreadAdder</receiver>
   <slot>startSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartAll</sender>
   <signal>clicked()</signal>
   <receiver>UIThreadAdder</receiver>
   <slot>startAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btClearSelected</sender>
   <signal>clicked()</signal>
   <receiver>UIThreadAdder</receiver>
   <slot>clearSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>627</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startAll()</slot>
  <slot>startSelected()</slot>
  <slot>clearAll()</slot>
  <slot>clearSelected()</slot>
  <slot>selectAll()</slot>
  <slot>selectNone()</slot>
 </slots>
</ui>
